lib/rules/at-rule-semicolon-space-before/README.md

```diff
@@ -0,0 +1,45 @@
+# at-rule-semicolon-space-before
+
+Require a single space or disallow whitespace before the semicolons of regular (non-nested) at-rules.
```

Comment:
"Require a single space or disallow whitespace after the semicolons of at-rules."

There's doesn't appear to be anything in the rule that avoids checking nested rules. Let's check _all_ at-rules to keep it consistent with [`at-rule-semicolon-newline-after`](https://github.com/stylelint/stylelint/tree/master/lib/rules/at-rule-semicolon-newline-after)